Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
 inherits = "release"
 codegen-units = 1
 lto = true
+strip = "symbols"
```

Comment:
Do we use `--production` in the CI somewhere or is this manual exeuction only?
If so, is the release checklist adapted so we don't mistakenly publish a `--release` build

Reply:
Our CI has been switched to `production` profile.